### ISSUE TITLE
Undefined symbols for architecture arm64

### ISSUE THREAD

**Issue by chillin0531** [zendesk/zendesk_sdk_ios#499]
SDK version: 2.3.1
Xcode version: 10.1
iOS or Simulator: iPhone / iPad
iOS Version: iOS 12

Created a new Objective-C app in Xcode and integrated the Zendesk Support V2 SDK by following the steps provided. When I build the project getting the following error.

`Undefined symbols for architecture arm64:
  "_OBJC_CLASS_$_ZDKSupport", referenced from:
      objc-class-ref in AppDelegate.o
  "_OBJC_CLASS_$_ZDKZendesk", referenced from:
      objc-class-ref in AppDelegate.o
ld: symbol(s) not found for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
`

I have added security and mobile coreservices framework in addition.

**Comment by RonanMcH** [comment]
Hi @chillin0531 ,
Did you integrate the SupportSDK using Cocoapods or manual integration steps? I haven't managed to recreate this issue locally. I can see in the repo that the arm64 swift modules are there for the ZendeskProviderSDK which contain the `ZDKZendesk` symbol. Can you check this location `ZendeskProviderSDK.framework/Modules/ZendeskProviderSDK.swiftmodule` and let us know if you see the arm64 module?
Thanks,
Ronan.

**Comment by chillin0531** [comment]
@RonanMcH Thanks for your reply. 
I have integrated the SDK manually. Yes I can able to locate the arm 64 module in `ZendeskProviderSDK.framework/Modules/ZendeskProviderSDK.swiftmodule` 
As of now I have copied the frameworks from the sample app [here](ZendeskProviderSDK.framework/Modules/ZendeskProviderSDK.swiftmodule) and its working now without any errors.
Still can't figure it out the cause of this issue, when I add the frameworks from the SDK repo.
